webapp/client/src/components/FormFieldIdNr.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabelForSeparatedFields from "./FieldLabelForSeparatedFields";
+import FormFieldSeparatedField from "./FormFieldSeparatedField";
+import {
+  baselineBugFix,
+  numericInputMask,
+  numericInputMode,
+} from "../lib/fieldUtils";
+
+function FormFieldIdNr({
+  fieldName,
+  fieldId,
+  values,
+  required,
+  autofocus,
+  label,
+  details,
+  errors,
+}) {
+  const labelComponent = (
+    <FieldLabelForSeparatedFields {...{ label, fieldId, details }} />
+  );
+
+  const extraFieldProps = {
+    ...numericInputMode,
+    ...numericInputMask,
+    ...baselineBugFix,
+  };
+
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+        labelComponent,
+      }}
+      hideLabel
+      hideErrors
+      render={() => (
+        // Separated fields ignore field class names
+        <FormFieldSeparatedField
+          {...{
+            fieldName,
+            labelComponent,
+            errors,
+            details,
+            extraFieldProps,
+            fieldId,
+            values,
+            required,
+          }}
+          autofocus={autofocus || Boolean(errors.length)}
+          inputFieldLengths={[2, 3, 3, 3]}
+        />
+      )}
+    />
+  );
+}
+
+FormFieldIdNr.propTypes = {
+  fieldId: PropTypes.string.isRequired,
+  fieldName: PropTypes.string.isRequired,
+  errors: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
I suppose it could be optional with a default value of `[]`…? I don't think it matters though, we're always going to set it to _some_ list from Python (which may be empty).